src/main/webapp/map.html

```diff
@@ -0,0 +1,59 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Hello Google Maps</title>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAIJxmuluUxbW2r2rZgOebw7B0U1rRtrvA"></script>
+    <script>
+      /** Creates a map that shows landmarks around Google. */
+      function createMap(){
+        const map = new google.maps.Map(document.getElementById('map'), {
+          center: {lat: 37.422403, lng: -122.088073},
+          zoom: 15
+        });
+
+        addLandmark(map, 37.423829, -122.092154, 'Google West Campus',
+            'Google West Campus is home to YouTube and Maps.')
+        addLandmark(map, 37.421903, -122.084674, 'Stan the T-Rex',
+            'This is Stan, the T-Rex statue.')
+        addLandmark(map, 37.420919, -122.086619, 'Permanente Creek Trail',
+            'Permanente Creek Trail connects Google to a system of bike trails.');
+      }
+
+      /** Adds a marker that shows an info window when clicked. */
+      function addLandmark(map, lat, lng, title, description){
+        const marker = new google.maps.Marker({
+          position: {lat: lat, lng: lng},
+          map: map,
+          title: title
+        });
+        const infoWindow = new google.maps.InfoWindow({
+          content: description
+        });
+        marker.addListener('click', function() {
+          infoWindow.open(map, marker);
+        });
+      }
+    </script>
+    <style>
+      #map{
+        width: 500px;
+        height: 500px;
+        border: thin solid black;
+      }
+    </style>
+  </head>
+  <body onload="createMap();">
+    <nav>
+      <ul id="navigation">
+        <li><a href="/">Home</a></li>
+        <li><a href="/aboutus.html">About Our Team</a></li>
+        <li><a href="/stats.html">Stats</a></li>
+        <li><a href="/map.html">Map</a></li>
+      </ul>
+    </nav>
+    <h1>Fun Food Statistics</h1>
+    <p>Interesting facts about food!haha!</p>
+    <div id="map"></div>
+  </body>
+  </html>
```

Comment:
the </html> should be one more indentation in I think

Reply:
To clarify, do you mean one level forward or backward @dayhahm? I suspect backwards but I just wanna make sure.

Reply:
Oops I did mean backwards. Thanks for asking for the clarification